.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @nikolamajcen @ZvonimirMedak
```

Comment:
Honestly, I'm totally fine that you set only yourself here as a maintainer. I could be added manually any time you need me 👍